src/incidencehandler.cpp

```diff
@@ -123,24 +123,6 @@ KCalendarCore::Incidence::Ptr IncidenceHandler::incidenceToExport(KCalendarCore:
         }
     }
 
-    // The default storage implementation applies the organizer as an attendee by default.
-    // Undo this as it turns the incidence into a scheduled event requiring acceptance/rejection/etc.
```

Comment:
Wondering when reading the comment if the right thing to do for old incidences was to add the organiser as attendee...

What may happen now when an old incidence is upsynced (for a time change reason for instance), from the comment, is that this event will trigger sending mails to attendees. Maybe that's the right thing to do after all...

Looking at the history, the commit introducing the comment was not explaining the possible issue. As far as I remember, there was an issue like that in Google sync, and that's why the organizer as attendee was always removed there. Maybe it was done here for the same reason ? Something on TJC about Google sending mails.

Maybe valuable to investigate a bit there to see if we won't have any unexpected consequences.

Reply:
Got it : https://together.jolla.com/question/189920/google-calendar-sync-auto-invites-me/

Reply:
Suppose
- newly synced events should be good, organizer and attendees are what are set on server side or explicitly by the app when creating an event
- old events can trigger sync only if modified and if there's an organizer it should happen only for events created by the user
- worst that might happen is that the user send email for him/herself

If so, wouldn't sound too bad.